test_scripts/Polices/Validation_of_PolicyTables/248_ATF_pt_snapshot_path_is_incorrect.lua

```diff
@@ -89,26 +91,27 @@ function Test:Precondition_StopSDL()
 end
 
 function Test:Precondition_ReloadNewINIFile()
-  oldPathToPtSnapshot = self.changePtsPathInSdlIni(INCORRECT_LINUX_PATH_TO_POLICY_SNAPSHOT_FILE)
+  self.changePtsPathInSdlIni(INCORRECT_LINUX_PATH_TO_POLICY_SNAPSHOT_FILE)
 end
 
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
+function Test.TestStep_start_sdl()
+  StartSDL(config.pathToSDL, config.ExitOnCrash)
+  os.execute("sleep 5")
+end
 
 function Test:TestStep_VerifySDL_Stops()
-  --TODO(istoimenova): Should be checked when ATF problem is fixed with SDL crash
-  EXPECT_HMINOTIFICATION("BasicCommunication.OnSDLClose")
-  local result = testCasesForPolicySDLErrorsStops:CheckSDLShutdown(self)
-  if (result == false) then
-    self:FailTestCase("Error: SDL doesn't stop.")
-  end
+  testCasesForExternalUCS.checkSDLStatus(self, sdl.STOPPED)
 end
 
-function Test:TestStep_CheckSDLLogError()
-  local result = testCasesForPolicySDLErrorsStops.ReadSpecificMessage("PathToSnapshot is not correct")
-  if (result == false) then
-    self:FailTestCase("Error: message 'PathToSnapshot is not correct' is not observed in smartDeviceLink.log.")
-  end
+function Test.TestStep_CheckSDLLogError()
```

Comment:
@istoimenova, Please add check on "BasicCommunication.OnSDLClose" in order to conform to other scripts